src/helix-ui/elements/HXSearchElement.less

```diff
@@ -1,6 +1,8 @@
 @import (reference) "mixins";
 @import "./HXElement.less";
 
+// TODO: add CSS custom props for theming hooks
+
```

Comment:
This will take place when we refactor the Search component. Updates were too extensive to remain in scope for this PR.